packages/lit-dev-content/site/external-search-data/videos.html

```diff
@@ -0,0 +1,6 @@
+---
+permalink: external-search-data/videos.json
+---
+
+{% if not env.DEV %}{{ videos | videosToAlgoliaRecords | dump | safe }}{%endif
+%}
```

Comment:
Is it ok to split this over to the next line?

Reply:
it seems to work. Prettier seems to have formatted it that way. will change though